### ISSUE TITLE
Dask runner doesn't support ElementwiseProblem

### ISSUE THREAD

**Issue by grandrew** [anyoptimization/pymoo#425]
It emits pickling errors when:

```py
from pymoo.algorithms.soo.nonconvex.ga import GA
from pymoo.optimize import minimize
from pymoo.core.problem import DaskParallelization
from pymoo.core.problem import ElementwiseProblem
# from pymoo.core.problem import Problem
import numpy as np

from dask.distributed import Client
client = Client("... scheduler address ...")
client.restart()
print("DASK STARTED")

# initialize the thread pool and create the runner
runner = DaskParallelization(client)

class MyProblem(ElementwiseProblem):
# class MyProblem(Problem):

    def __init__(self, **kwargs):
        super().__init__(n_var=10, n_obj=1, n_ieq_constr=0, xl=-5, xu=5, **kwargs)

    def _evaluate(self, x, out, *args, **kwargs):
         out["F"] = np.sum(x ** 2, axis=1)

# define the problem by passing the starmap interface of the thread pool
problem = MyProblem(elementwise_runner=runner)

res = minimize(problem, GA(), termination=("n_gen", 200), seed=1)
print('Threads:', res.exec_time)

client.close()
print("DASK SHUTDOWN")
```

the error is:

```
pymoo/core/problem.py", line 57, in __getstate__
    state.pop("client")
KeyError: 'client'

```

lines involved:

```py

class DaskParallelization:

    def __init__(self, client) -> None:
        super().__init__()
        self.client = client

    def __call__(self, f, X):
        jobs = [self.client.submit(f, x) for x in X]
        return [job.result() for job in jobs]

    def __getstate__(self):
        state = self.__dict__.copy()
        state.pop("client")    # Exception here
        return state


```

**Comment by cyrilpic** [comment]
Hi! This issue has been fixed in https://github.com/anyoptimization/pymoo/commit/9a38ad893ff9896d1fd2eb4e22b93e72b5ed051c (#363). I think it is not yet in a packaged version of `pymoo`, but you can use it by installing from GitHub.

**Comment by blankjul** [comment]
Thanks @cyrilpic. Does the current release on GitHub resolve the issue?

**Comment by cyrilpic** [comment]
This fix was not part of pymoo 0.6.0.1 and I cannot find a more recent release. Installing from Github's main branch works.
